FHICORC.Core/Services/Model/SmartHealthCardModel/Shc/SmartHealthCardWrapper.cs

```diff
@@ -21,5 +23,22 @@ public class SmartHealthCardWrapper : ITokenPayload
         public DateTime? ExpiredDateTime() => SmartHealthCard.ExpirationDate;
 
         public DateTime? IssueDateTime() => SmartHealthCard.IssuanceDate;
+
+        public TokenValidateResult Validate()
+        {
+            if (ExpiredDateTime() != null && ExpiredDateTime() < DateTime.Today)
```

Comment:
Should you account for UTC time here, or not? Because DateTime.Today for timezones with positive offset to UTC might invalidate the valid SHCs in this case.

Reply:
Good point! Canged to Now to include time. 
And used UtcNow with ExpiredDateTime.ToUniversalTime(), so both are in UTC.